### DISCUSSION TITLE
Tailwind Css error after deploying to vercel it disrpts the ui

### DISCUSSION THREAD

**Discussion by akhilpamwani** [vercel/next.js#37308] [Help]
### Verify canary release

- [X] I verified that the issue exists in Next.js canary release

### Provide environment information

{
  "name": "hunting-coder",
  "private": true,
  "scripts": {
    "dev": "next dev",
    "build": "next build",
    "start": "next start",
    "lint": "next lint"
  },
  "dependencies": {
    "aos": "^2.3.4",
    "axios": "^0.26.0",
    "cssnano": "^5.1.8",
    "next": "^12.1.6",
    "node-fetch": "^3.2.3",
    "react": "17.0.2",
    "react-dom": "17.0.2",
    "react-router-dom": "^6.3.0",
    "react-toastify": "^8.2.0",
    "sass": "^1.49.7",
    "sharp": "^0.30.1",
    "tw-elements": "^1.0.0-alpha12"
  },
  "devDependencies": {
    "@babel/core": "^7.18.2",
    "autoprefixer": "^10.4.2",
    "eslint": "8.8.0",
    "eslint-config-next": "12.0.10",
    "postcss": "^8.4.6",
    "postcss-import": "^14.1.0",
    "tailwindcss": "^3.0.22"
  }
}

### What browser are you using? (if relevant)

Microsoft  Edge

### How are you deploying your application? (if relevant)

Vercel

### Describe the Bug

After the deployment to vercel of next it looks like = https://code-hustler.vercel.app/ 
This is not the website that i created.

Github Repo=== https://github.com/akhilpamwani/CodeHustler

### Expected Behavior

Ui Disrupts.

### To Reproduce

Step:1 Go to https://code-hustler.vercel.app/
Step 2 You will ui difference.

**Comment by BkSouX** [comment]
i have exactly the same issue when I move from next 12.1.4 to next 12.1.6
it works with next dev but when I do a next build and a next start, tailwind css is not loaded.

**Reply by akhilpamwani** [reply]
Is there any solution

**Reply by BkSouX** [reply]
im currently trying things but still not working

**Reply by BkSouX** [reply]
hi, my issue was this config :

`  pageExtensions: process.env["CC_SITE_ID"] ? ["static.tsx"] : ["server.tsx"],`

server.tsx extension is like a protected extension we cannot use. It is not mentioned anywhere.
